rudder-web/src/main/scala/com/normation/rudder/web/snippet/administration/PropertiesManagement.scala

```diff
@@ -694,7 +681,50 @@ class PropertiesManagement extends DispatchSnippet with Loggable {
     ) apply (xml ++ Script(Run("correctButtons();") & check()))
   }
 
+  def apiComptabilityMode = { xml : NodeSeq =>
 
-}
+    //  initial values, updated on successfull submit
+    var initApiMode = configService.api_compatibility_mode()
+
+    // form values
+    var apiMode  = initApiMode.getOrElse(true)
 
+    def submit = {
+      configService.set_api_compatibility_mode(apiMode).foreach(updateOk => initApiMode = Full(apiMode))
+
+      // start a promise generation, Since we check if there is change to save, if we got there it mean that we need to redeploy
+      S.notice("apiModeMsg", "Api compatibility mode changed")
+      check()
```

Comment:
why should it start a generation ? it has only impact on the AP, not the generated promises